vivisect/__init__.py

```diff
@@ -630,13 +630,14 @@ def initWorkspaceClient(self, remotevw):
         self.server.vprint('%s connecting...' % uname)
         wsevents = self.server.exportWorkspace()
         self.importWorkspace(wsevents)
-        self._snapInAnalysisModules()
         self.server.vprint('%s connection complete!' % uname)
 
         thr = threading.Thread(target=self._clientThread)
         thr.setDaemon(True)
         thr.start()
 
+        self._snapInAnalysisModules()
```

Comment:
we're already in a race condition.  that's what this PR is addressing.
believe it or not, the code works right now...  most of the time.

Reply:
So that's kinda what I'm getting at. If we're gonna fix it, let's fix it fully, not mostly. And I think we can do that with some of the constructs in the threading library.